gnosis/dex-react
# Move responsability from row to widget

Fixes #118 

Moves responsibility for updating the data to the `DepositWidget` from row/form.
Expect now the row logic to be much smaller.

**Includes**:
- [X] Models the highlight/select as props for row. Also the events for deposit, withdraw, claim or enable. This way is the parent responsibility now to provide them
- [X] Implement highlight/select logic in `DepositWidget` instead of using the hook

**Aditionally**:
- [X] Adapt tests to the new properties. Add some tests. Small refactor on them.
- [X] Define const for highlight time (used in hook and in widget for now)
- [X] Small refactor for `withdrawing` prop. Renaming it to `claiming` to be coherent with the UI

**Test**:
- Run unit testing
- Check: https://pr120--dexreact.review.gnosisdev.com
